docs-website/pages/cli-run-plugin.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Run a plugin
+section: cli
+chapter: Commands
+permalink: /cli/run-plugin
+
+order: 200
+---
+
+SketchTool can tell Sketch to launch and run a plugin. This is very useful if you're testing a plugin on a continuous integration system, or you need to automate boring tasks.
+
+Imagine we have a plugin with this code:
+
+```
+context.document.showMessage("Remote plugin running!")
+```
+
+We save it as 'Remote Plugin' from the _Run Script…_ panel, and run this:
+
+```
+sketchtool run ~/Library/Application\ Support/com.bohemiancoding.sketch3/Plugins/Remote\ Plugin.sketchplugin com.bohemiancoding.sketch.runscriptidentifier
+```
+
+Note that `com.bohemiancoding.sketch.runscriptidentifier` is the default command identifier Sketch uses when saving a plugin, but it may be different in your case. If you just want to run the first command in the bundle, you can use `""` instead of an identifier.
```

Comment:
I’m a little unclear how to run the first command in the bundle.

is it `com.bohemiancoding.sketch.""`